webui/react/src/components/ClusterMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { ClusterMessage } from 'stores/determinedInfo';
+
+import css from './ClusterMessage.module.scss';
+
+interface Props {
+  message?: ClusterMessage;
+}
+
+const ClusterMessageBanner: React.FC<Props> = ({ message }) => {
+  return message ? (
+    <div className={css.base}>
+      <span>
+        <span className={css.clusterMessageLabel} data-testid="admin-msg">
+          Message from Admin:
+        </span>{' '}
+        <span data-testid="cluster-msg">{message.message}</span>
```

Comment:
qq: do we wanna show empty string when `message.message` is empty string? 

<img width="286" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/2143bc02-5fe5-4317-abb5-7e8c117c2d41">

Reply:
Ah, good catch. I think we should only show the banner when the message has length > 0.